recipes/pyshex/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pyshex" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyShEx-{{ version }}.tar.gz
+  sha256: 3c5c4d45fe27faaadae803cb008c41acf8ee784da7868b04fd84967e75be70d0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - generate_parser = pyshex.shex_evaluator:evaluate_cli
```

I don't see that `generate_parser`. This one is in the metadata though:

```suggestion
    - shexeval = pyshex.shex_evaluator:evaluate_cli
```

---

Oops, copy paste error, sorry!